modules/govuk_env_sync/files/govuk_env_sync.sh

```diff
@@ -358,7 +358,28 @@ function dump_postgresql {
 }
 
 function output_restore_sql {
-  pg_restore -j 2 "${dumpfile}" | sed -r "${sed_cmds}"
+
+  PG_RESTORE_VERSION='13.4'
+  DUMPFILE_VERSION=$(file "${dumpfile}" | awk '{print $NF}')
+
+  case $DUMPFILE_VERSION in
+    v1.13-0)
+      PG_RESTORE_VERSION='9.6.22'
+      ;;
+    v1.14-0)
+      PG_RESTORE_VERSION='13.4'
+      ;;
+    *)
+      >&2 echo "${DUMPFILE_VERSION} is not a supported dump file version"
+      exit 1
+      ;;
+  esac
+  sudo docker run --rm --net=host  \
+	  -v "${tempdir}:/tmp/" \
+	  -v "/root/.pgpass:/tmp/.pgpass" -e PGPASSFILE=/tmp/.pgpass \
+	  "postgres:$PG_RESTORE_VERSION" \
+	  pg_restore -j 2  -f "/tmp/restore.sql" "/tmp/${filename}" | sed -r "${sed_cmds}"	
```

Comment:
I wonder if we want restore.sql to be project specific so it doesn't risk any clashes or using the wrong file on a db_admin talking to multiple databases?